### DISCUSSION TITLE
40회차 반성문

### DISCUSSION THREAD

**Discussion by amaran-th** [BDD-CLUB/challenge-every-week-posting#386] [6. 반성문]
주제는 생각을 해두었는데 저번 주 내내 상태가 안 좋아서 글을 쓸 정신이 없었네요...
늘 컨디션을 일정하게 유지하는 것도 쉬운 일은 아닌 것 같습니다.
사실 글을 쓰는 데는 하루 이틀이면 충분한데, 상황에 따라 그만한 시간을 내기 어렵거나 맘에 여유가 없어서 자꾸 챌린지를 실패하게 되는 것 같아요.
이번 주부터는 빨리 멘탈을 회복하고 다시 글쓰기에 도전해보겠습니다.
모두 파이팅~~

**Comment by stopmin** [comment]
시간의 여유가 없는 것보다, 마음에 여유가 없는 것이 글 쓰는거나 무언가를 하는데 더 큰 비용을 소모하게 만드는 것 같아요🥲
같이 파이팅입니당 !! 🔥🔥🔥
